src/posts/2021-09-03-getting-started-with-orca.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Getting started with Orca screen reader on Gnome desktop environment on Ubuntu 20.04 LTS
+description: A quick overview of getting started with Orca on Gnome desktop environment on Ubuntu 20.04 LTS.
+category: Assistive technology
+author: Joseph mawa
```

Comment:
I noticed you capitalized "mawa" in `authors.json`, but it is lowercase here. I want to ensure your name is displayed the way you want it to - which form do you use?

Reply:
Thanks for catching this. It is supposed to be "Mawa".